src/hotspot/share/opto/chaitin.cpp

```diff
@@ -1640,6 +1657,41 @@ uint PhaseChaitin::Select( ) {
         }
       }
     }
+
+    Node* def = lrg->_def;
+    MachNode* mdef = lrg->is_singledef() && !lrg->_is_bound && def->is_Mach() ? def->as_Mach() : nullptr;
+    if (mdef != nullptr) {
+      int i = 1;
+      uint lrg_def = _lrg_map.find(def);
+      for (; i < mdef->num_opnds(); i++) {
+        if (Matcher::is_register_biasing_candidate(mdef, 1, i)) {
```

Comment:
`_copy_bias` and `_copy_bias2` initialization code is mostly a duplication. Please, extract it into a helper function.